Rock Paper Scissors/rock-paper-scissors.html

```diff
@@ -0,0 +1,111 @@
+            let playerChoice = userInput.charAt(0).toUpperCase() + userInput.slice(1).toLowerCase();
+
+            console.log(`Player: ${playerChoice}`)
+            console.log(`Computer: ${computerChoice}`)
+        
+
+            if (playerChoice == computerChoice) {
+                console.log(`Its a draw! ${playerChoice} equals ${computerChoice}`);
+                
+            }
+            else {     
+                switch (playerChoice) {
+                    case "Rock":  
+                        if (computerChoice == "Scissors") {  
+                            userScore++;
+                            console.log(`You win! ${playerChoice} beats ${computerChoice}`);
+                        }
+                        else { 
+                            computerScore++;
+                            console.log(`You lose! ${computerChoice} beats ${playerChoice}`);
+                        }
+                        break;
+                    case "Paper":
+                        if (computerChoice == "Rock") { 
+                            userScore++;
+                            console.log(`You win! ${playerChoice} beats ${computerChoice}`);
+                        }
+                    else {
+                        computerScore++;
+                        console.log(`You lose! ${computerChoice} beats ${playerChoice}`);    
+                        }
+                        break;
+                    case "Scissors":
+                        if (computerChoice == "Paper") { 
+                            userScore++;
+                            console.log(`You win! ${playerChoice} beats ${computerChoice}`);
+                        }    
+                        else {
+                            computerScore++;
+                            console.log(`You lose! ${computerChoice} beats ${playerChoice}`);    
+                        }
+                        break;    
+                    default : {
+                        console.log("Enter a proper value");
+                        console.log(playerChoice);
+                        playRound();
+                        
+                        }
+       
+                }
+    
+        }
+        }
+
+        function finalResult() {
+            if (computerScore > userScore){
+                
+                console.log("You lose the game.")
+            } 
+            
+            else if (userScore > computerScore){
+                console.log("You win the game.");
+            }
+
+            else {
+                console.log("Its a draw");
+            }
+        }
+
+        function game() {
+ 
+            for (let i = 0; i<5; i++ ) {
+        
+            playRound();
+            console.log(`Your score is ${userScore}.`);
+            console.log(`Computer's score is ${computerScore}.`);
+            
+            }
+        
+        
+            
+        }
+        
+    game();
+    finalResult();
```

Comment:
The finalResult function is part of the game. It should not be left out there.

Reply:
ok refactored